docs/how-tos/move-robot-arm.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Access and move a robot arm"
+linkTitle: "Move a robot arm"
+weight: 70
+type: "docs"
+tags: ["arm", "components"]
+images: ["/tutorials/motion/preview.jpg"]
+description: "Access and control one of the most fundamental systems in robotics: A robotic arm."
+aliases:
+  - /tutorials/motion/accessing-and-moving-robot-arm
+  - /tutorials/motion/
+languages: ["Python", "Go"]
+viamresources: ["arm"]
+platformarea: ["mobility"]
+level: "Intermediate"
+date: "2024-10-31"
+# updated: ""  # When the tutorial was last entirely checked
+cost: "8400"
+---
+
+In this guide you'll use Viam's Python or Go SDKs to access data about the state of your robot arm and issue movement commands.
+
+{{<imgproc src="/tutorials/motion/access_01_xarm6.png" resize="500x" declaredimensions=true alt="A picture of the UFACTORY xArm 6." style="width: 150px" class="fill alignleft" >}}
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Access the arm](#access-the-arm)
+1. [Move the arm](#move-the-arm)
+
+{{< /alert >}}
+
+{{< alert title="Caution" color="caution" >}}
+Be careful when instructing robot arms to move.
+Before running any code, ensure your robotic arm has enough space and that there are no obstacles.
+Also pay attention to your surroundings, double-check your code for correctness, and make sure anyone nearby is aware and alert before issuing commands to your machine.
+{{< /alert >}}
+
+## Access the arm
```

Comment:
Are you sure it's wise to add in the info about a fake arm? I had it in there but decided to remove it because you can't see any of the movement commands, it seems a bit pointless.